include/rear-panel/core/bin_msg_ids.hpp

```diff
@@ -18,6 +18,8 @@ enum class MessageType : uint16_t {
     RELEASE_SYNC_REQUEST = 0x0A,
     ESTOP_STATE_CHANGE = 0x0B,
     ESTOP_BUTTON_DETECTION_CHANGE = 0x0C,
+    DOOR_SWITCH_STATE_REQUEST = 0x0D,
+    DOOR_SWITCH_STATE_RESPONSE = 0x0E,
```

Comment:
we should probably name this something that makes it clearer it can be asynchronously sent by the rear panel